integration-tests/src/test/resources/features/end2endtestit.feature

```diff
@@ -4,26 +4,26 @@ Feature: End to End Bridge integration tests
     Given get list of Bridge instances returns HTTP response code 401
 
 
-  Scenario: Bridge is created and in available state
+  Scenario: Bridge is created and in ready state
     Given get list of Bridge instances with access token doesn't contain randomly generated Bridge
 
     When create a Bridge with randomly generated name with access token
     And get list of Bridge instances with access token contains Bridge with randomly generated name
-    And get Bridge with access token exists in status "AVAILABLE" within 2 minutes
+    And get Bridge with access token exists in status "ready" within 2 minutes
     And delete a Bridge
 
     Then the Bridge doesn't exists within 2 minutes
     And the Ingress is Undeployed within 1 minute
     And the Manager Metric 'manager_bridge_status_change_total{status="PROVISIONING",}' count is at least 1
-    And the Manager Metric 'manager_bridge_status_change_total{status="AVAILABLE",}' count is at least 1
+    And the Manager Metric 'manager_bridge_status_change_total{status="READY",}' count is at least 1
```

Comment:
Is there some convention when to use "ready" vs "READY" please?

Reply:
`ready` is only in the responses for the user's api.